source/Octopus.TestPortForwarder/PortForwarder.cs

```diff
@@ -67,6 +67,7 @@ private void Start()
             }
 
             listeningSocket ??= new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
+            listeningSocket.NoDelay = true;
```

Comment:
In our test portforwarder we never intended it to by default add buffering.

Reply:
When merged, it would be worth bumping the version of the Proxy in Tentacle as well

Reply:
Good idea